charts/synthetics-minion/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v2
 name: synthetics-minion
 description: New Relic Synthetics Containerized Private Minion (CPM)
 type: application
-version: 1.0.38
+version: 1.0.40
```

The current version is 1.0.40 so we'll want to bump this to `1.0.41`. The `appVersion` can remain the same though

```suggestion
version: 1.0.41
```